docs/cosmology/units.rst

```diff
@@ -0,0 +1,86 @@
+.. _astropy-cosmology-units-and-equivalencies:
+
+************************************
+Cosmological Units and Equivalencies
+************************************
+
+.. currentmodule:: astropy.cosmology.units
+
+This package defines and collects cosmological units and equivalencies.
+Many of the units and equivalencies are also available in the
+:mod:`astropy.units` namespace.
+
+We suggest importing this units package as
+
+    >>> import astropy.cosmology.units as cu
+
+
+If you want to access all units through :mod:`astropy.units`, then this units
+module can be added to the astropy.units namespace using
+:func:`~astropy.units.add_enabled_units`.
+
+    >>> u.add_enabled_units(cu)  # doctest: +SKIP
+
+
+About the Units
+===============
+
+.. _littleh-and-H0-equivalency:
+
+Reduced Hubble Constant and "little-h" Equivalency
+--------------------------------------------------
+
+The dimensionless version of the Hubble constant — often known as "little h" —
+is a frequently used quantity in extragalactic astrophysics. It is also widely
+known as the bane of beginners' existence in such fields (See e.g., the title
+of `this paper <https://doi.org/10.1017/pasa.2013.31>`__, which also provides
+valuable advice on the use of little h). ``astropy`` provides an equivalency
+that helps keep this straight in at least some of these cases, by providing a
+way to convert to/from physical to "little h" units.
+
+.. EXAMPLE START: Using the "little h" Equivalency
+
+To convert to or from physical to "little h" units:
+
+    >>> import astropy.units as u
+    >>> H0_70 = 70 * u.km/u.s / u.Mpc
+    >>> distance = 70 * (u.Mpc/u.littleh)
+    >>> distance.to(u.Mpc, u.with_H0(H0_70))  # doctest: +FLOAT_CMP
+    <Quantity 100.0 Mpc>
+    >>> luminosity = 0.49 * u.Lsun * u.littleh**-2
+    >>> luminosity.to(u.Lsun, u.with_H0(H0_70))  # doctest: +FLOAT_CMP
+    <Quantity 1.0 solLum>
+
+Note the unit name ``littleh``: while this unit is usually expressed in the
+literature as just ``h``, here it is ``littleh`` to avoid confusion with
+"hours."
+
+If no argument is given (or the argument is `None`), this equivalency assumes
+the ``H0`` from the current default cosmology:
+
+    >>> distance = 100 * (u.Mpc/u.littleh)
+    >>> distance.to(u.Mpc, u.with_H0())  # doctest: +FLOAT_CMP
+    <Quantity 147.79781259 Mpc>
+
+This equivalency also allows a common magnitude formulation of little h
+scaling:
+
+    >>> mag_quantity = 12 * (u.mag - u.MagUnit(u.littleh**2))
+    >>> mag_quantity  # doctest: +FLOAT_CMP
+    <Magnitude 12. mag(1 / littleh2)>
+    >>> mag_quantity.to(u.mag, u.with_H0(H0_70))  # doctest: +FLOAT_CMP
+    <Quantity 11.2254902 mag>
+
+.. EXAMPLE END
+
+
+|
```

Comment:
Why these `|`?

Reply:
When I look at RTD, our sphinx template places sections too close for my taste. I think the new scipy user docs fix this, but until (if) we migrate it's necessary to manually increase the spacing.

Reply:
Obviously not the biggest deal, but my tendency would be to not include by-hand spacing; it often is browser/OS/taste dependent, and we should just fix it in the template we use.

Reply:
Is there a timeline for adopting the scipy template?
I can take a look at the astropy one...

Reply:
Never mind. I reverted the added spaces.